front/src/modules/trainschedule/components/ManageTrainSchedule/helpers/formatTimetableItemPayload.ts

```diff
@@ -46,6 +46,6 @@ export function formatPacedTrainPayload(
       time_window: osrdconf.timeWindow.toISOString(),
       interval: osrdconf.interval.toISOString(),
     },
-    exceptions: [],
+    exceptions: osrdconf.addedExceptions,
```

Comment:
Don't we need to convert dates to strings here via `toISOString()`? I'm curious why TypeScript doesn't complain here.

Reply:
we do, it’s fixed !